### PR TITLE
[#1194] Fixed popup layout with smaller videos

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1194-popup-template-positions-of-pop-ups-change-when-media-is-switched

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/8bd0f1518d70198/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8bd0f1518d70198/output.txt

Total script time: 0.66 mins

Server is running on http://pr469.butyr.org/templates/test.html

File: templates/supported/popup/style.css [hunk 1]
```diff
@@ -23,6 +23,17 @@ video {
   position: relative;
   margin: 1em auto;
   width: 768px;
+  height: 432px;
+}
+
+#video {
```
Comment (cadecairos) [inline templates/supported/popup/style.css]: instead of two declarations just do:

```
#video, #video iframe {
    height: 100%;
    width: 100%;
}
```

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/efb8f7ba74ce88b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/29b32207afa87f3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/efb8f7ba74ce88b/output.txt

Total script time: 0.50 mins

Server is running on http://pr469.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/29b32207afa87f3/output.txt

Total script time: 0.41 mins

**Pull request merged**